src/cli.rs

```diff
@@ -23,6 +23,8 @@ pub enum Command {
     Build(Build),
     /// Start your app previously built with the build command
     Start(Start),
+    /// Stop your app previously started with the start command
```

Comment:
```suggestion
    /// Stops your app previously started with the start command
```

Reply:
I'm not sure of this one. It's like the other commands.

The CLI are mostly explaining what "you" do using the command and what the command does.